courses-app/src/components/CreateCourse/index.js

```diff
@@ -0,0 +1,2 @@
+import CreateCourse from './CreateCourse';
+export default CreateCourse;

```

Comment:
could be as one line

`export * from './CreateCourse'`

Reply:
fixed over all porject